### PR TITLE
Detect Ubuntu version

### PR DESCRIPTION
**Pull request by fpoli**

### CODE REVIEW

**Comment by fpoli** [comment]
Why isn't CI running?

File: src/util/Platform.ts [hunk 1]
```diff
@@ -20,3 +21,61 @@ export const currentPlatform: Platform | null = (() => {
...
+    const releaseVersion = releaseDetails.version_id;
+    if (releaseVersion == undefined) {
+        console.log("The Ubuntu release version is undefined");
+        return UbuntuVersion.Unknown;
+    }
+    if ((Object.values(UbuntuVersion) as unknown[]).indexOf(releaseVersion) >= 0) {
+        return releaseVersion as UbuntuVersion;
+    } else {
+        console.log(`Unknown Ubuntu release version: ${releaseVersion}`);
+        return UbuntuVersion.Unknown;;
```
Comment (ArquintL) [inline src/util/Platform.ts:79]:
```suggestion
        return UbuntuVersion.Unknown;
```

File: src/util/Platform.ts [hunk 2]
```diff
@@ -20,3 +21,61 @@ export const currentPlatform: Platform | null = (() => {
...
+        // Split the line into an array of words delimited by "=" and remove quotes
+        const words = line.split("=");
+        if (words.length > 1) {
+            releaseDetails[words[0].trim().toLowerCase()] = words[1].trim().replace(/^"(.*)"$/, '$1');
+        }
+    });
+    return releaseDetails
+}
+
+export async function readUbuntuVersion(): Promise<UbuntuVersion | null> {
```
Comment (ArquintL) [inline src/util/Platform.ts:60]: I would add a short documentation mentioning that `null` is returned if the OS does not seem to be Ubuntu

**Review by ArquintL — APPROVED** [review]
only two minor comments / suggestions

**Pull request merged**